lib/techniqueAttributeForParameter.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
 module.exports = techniqueAttributeForParameter;
 
+/**
+ * Retrieves the primitive attributes that has a given parameter name.
+ *
+ * @param {Object} technique A javascript object containing a glTF technique.
+ * @param {String} parameterName The search string for parameter.
+ * @returns {Array.<string>} The primitive attribute with matching paramerterName.
```

Comment:
This one I got from [here](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/getPrimitiveAttributeSemantics.js#L13). Does this need correcting as well?